nginx.master.conf

```diff
@@ -4,26 +4,18 @@ include common/main.conf;
 http {
     # Without this, cosocket-based code in worker
     # initialization cannot resolve leader.mesos.
-    resolver 127.0.0.1:61053;
+    resolver 127.0.0.1:61053 ipv6=off;
 
     include common/http.conf;
 
     client_max_body_size 1024M;
 
-    upstream mesos {
-        server leader.mesos:5050;
-    }
-
     upstream marathon {
-        server master.mesos:8080;
-    }
-
-    upstream dcos_history_service {
-        server leader.mesos:15055;
+        server 127.0.0.1:8080;
```

Comment:
Turns out I am borked. My brain reproducibly ignored the fact that the `-    upstream dcos_history_service` line has minus and is read. I only consciously saw the change from leader.mesos to 127.0.0.1. Sorry!